src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -46,6 +46,22 @@ class ActivityFeed extends React.Component<Props, State> {
 
     feedContainer: null | HTMLElement;
 
+    componentDidMount() {
+        if (this.feedContainer) {
```

Comment:
Can we avoid duplicating this logic in mount, update, and render by moving the if-check and assignment to a helper method?

Reply:
addressed